frontend/src/components/Search.tsx

```diff
@@ -11,10 +11,13 @@ interface SearchResult {
 }
 
 const Search: React.FC = () => {
+  const{id} = useParams();
```

Comment:
should be `const {id} = ..` instead of `const{id} = .. `

please run lint:fix and format npm commands on your code once to fix formatting issues.